R/knitr-engine.R

```diff
@@ -767,22 +767,18 @@ eng_python_autoprint <- function(captured, options) {
 
   } else if (eng_python_is_altair_chart(value)) {
 
-    # set width if it's not already set
+    # set width and height if it's not already set
     # This only applies to Chart objects, compound charts like HConcatChart
     # don't have a 'width' or 'height' property attribute.
     # TODO: add support for propagating width/height options from knitr to
     # altair compound charts
     width <- py_get_attr(value, "width", TRUE)
```

Comment:
I am not sure if this line (and the corresponding one for `height`) are still needed, but I left them in for now.